azavea/pwd-waterworks-revealed
# Move zone O files to J. Rename zone K.

Connects #168 
Connects #167 

We want to remove zone O from the map and use the images from it in zone
J instead. I kept the unused images in the directory even though they
aren't being used for the moment. We also want to rename zone K at the
request of the client.

 * Move primary and secondary zone O images to zone J secondary images.
 * Deletes zone O files.
 * Re-title zone K to "South Gardens."
 * Regenerate quests.json

### To Test
 * Rebuild
 * Compare map to https://github.com/azavea/pwd-waterworks-revealed/wiki
 * Ensure zone O does not appear.
 * Enter zone J. Check that there are four secondary images.
 * Enter zone K. Ensure that the popup reads "You are in the South Gardens zone".
